website/settings/defaults.py

```diff
@@ -421,6 +421,8 @@ class CeleryConfig:
     task_high_queue = 'high'
     task_remote_computing_queue = 'remote'
     task_account_status_changes_queue = 'account_status_changes'
+    task_external_high_queue = 'external_high'
+    task_external_low_queue = 'external_low'
```

Comment:
I am fine with the names here, but double check with @adlius to see if he has better names for the queue.